### DISCUSSION TITLE
Image path error，after replace rect label to image  label in carousel, bootstrap v5

### DISCUSSION THREAD

**Discussion by gitnyj** [twbs/bootstrap#38628] [Q&A] [answered]
This is the original code in 5.3/examples/carousel/
```
<div class="carousel-item active">
  <svg class="bd-placeholder-img" width="100%" height="100%" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" preserveAspectRatio="xMidYMid slice" focusable="false"><rect width="100%" height="100%" fill="#777"/></svg>
  <div class="container">
    <div class="carousel-caption text-start">
      <h1>Example headline.</h1>
      <p>Some representative placeholder content for the first slide of the carousel.</p>
      <p><a class="btn btn-lg btn-primary" href="#">Sign up today</a></p>
    </div>
  </div>
</div>
```
I want to replace rect label to image  label，replace as follows：
`<image xlink:href="./images/slider01.jpg" width="100%" height="100%" />`
Browser prompt: GET httpp://localhost:8888/images/slider01.jpg 404 (Not Found)

webpack.config.js configuration：
```
{
  test: /\.(png|jpe?g|gif|svg)$/i, 
    type: "asset/resource",
      generator: {
    filename: 'assets/images/[name]_[hash:6][ext]'
  }
}
```

slider01.jpg path：src/images/

In index.html,if i use img label,like:`<img src="./images/slider01.jpg" class="d-block w-100" alt="..." />`,the image can be loaded and showed correctly.

So ,where is my mistake?I am a beginner,Thank you all for your help, master!

**Comment by ffoodd** [comment, marked as answer]
You'd probably better use a standard `<img>` tag, instead of modifying our docs' SVG. It's only a convenient placeholder that we use throughout the documentation.

There's no need to stick to SVG. The code examples in carousel's docs shows straight `<img>` tag, BTW.

**Reply by gitnyj** [reply]
Got it！Thanks a lot !
